links/src/utils/generateLinks.ts

```diff
@@ -10,8 +11,25 @@ type GenerateLinks = {
  */
 export const generateLinks: GenerateLinks = async () => {
 	const links = await getLinks();
+	const arr = Object.values(links);
+	let tree = totree(arr, 0);
 
-	// TODO: logic
+	fs.writeFileSync("./output.json", JSON.stringify(tree, null, 2));
 
 	return [];
```

Comment:
```suggestion
	return tree;
```

Looks like we can return tree as a result now?